constants/parameters.php

```diff
@@ -1,7 +1,7 @@
 <?php 
 const TEMPLATES_WE_PROCESS = array('citation', 'cite arxiv', 'cite book', 'cite document', 
                              'cite encyclopaedia', 'cite encyclopedia', 'cite journal', 'cite web');
-const TEMPLATES_WE_RENAME  = array('cite arxiv', 'cite book', 'cite document', 'cite journal', 'cite web');
+const TEMPLATES_WE_RENAME  = array('cite arxiv', 'cite book', 'cite document', 'cite journal', 'cite web', '');
```

Comment:
The intention here is: "if there is no template name, give one."

Otherwise `process` does not give a template name and the resulting Wiki reference is incomplete.

Reply:
How about set to cite web and then detect this in html/php?   IF still cite web, then probably nothing was done